Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -92,7 +94,12 @@ export class BotProject {
       // when re-index opened bot, file write may error
     }
     this.settings = await this.getEnvSettings('', false);
-    this.skills = await extractSkillManifestUrl(this.settings?.skill || []);
+    try {
+      this.skills = await extractSkillManifestUrl(this.settings?.skill || []);
+    } catch (_e) {
+      const notify = new Diagnostic('Accessing skill manifest url error', 'settings', DiagnosticSeverity.Error);
```

Comment:
Can we encode the url also here, to show exact which url?

Reply:
Sure